flashloan-detector/src/persistence.helper.spec.js

```diff
@@ -0,0 +1,179 @@
+const { PersistenceHelper } = require("./persistence.helper");
+const { existsSync, writeFileSync, unlinkSync } = require("fs");
+const fetch = require("node-fetch");
+
```

Comment:
```suggestion

const { Response } = jest.requireActual("node-fetch");
```

The tests somehow failed without this import when I ran them this time.

Reply:
Did you get any specific errors? They are passing locally on my machine when I run them.

Reply:
I'm getting a `ReferenceError: Response is not defined` when testing it on a different environment (GH Codespaces), while I don't, using my local machine. When not imported from `node-fetch` it seems that the import comes from the local Typescript module. Can you check the source of the import?